pkg/workflow/invocationcontextimpl.go

```diff
@@ -17,26 +18,32 @@ type InvocationContextImpl struct {
 	logger         *log.Logger
 }
 
+// GetWorkflowIdentifier returns the identifier of the workflow that is being invoked.
 func (ici *InvocationContextImpl) GetWorkflowIdentifier() Identifier {
 	return ici.WorkflowID
 }
 
+// GetConfiguration returns the configuration that is being used by the workflow engine.
```

Comment:
nitpick: here and in the following docs instead of
> ... that is being used by the workflow engine.

using 

> ... that can be used inside the workflow.

would be more accurate.